README.md

```diff
@@ -1,24 +1,160 @@
-# Solutions Repo Template
-
-Give your project a relevant name and description in this README.
+# Apollo Router Node.js proxy migration strategy
 
 **The code in this repository is experimental and has been provided for reference purposes only. Community feedback is welcome but this project may not be supported in the same way that repositories in the official [Apollo GraphQL GitHub organization](https://github.com/apollographql) are. If you need help you can file an issue on this repository, [contact Apollo](https://www.apollographql.com/contact-sales) to talk to an expert, or create a ticket directly in Apollo Studio.**
 
-## Installation
+## Run the demo
```

Comment:
Minor thing- I would note what `unleash` is as I wasn't aware of what it was at the start.

Reply:
ah yeah i'll add a small description at the top. i thought about just linking to an unwritten tech note on migration strategies, but i dunno when i'll get to that